setting/base.py

```diff
@@ -18,8 +18,8 @@
 EMBEDDING_MODEL_API_KEY = os.environ.get("EMBEDDING_MODEL_API_KEY", "ollama")
 
 # DB settings
-DATABASE_URI = os.environ.get("DATABASE_URI")
-SESSION_POOL_SIZE: int = os.environ.get("SESSION_POOL_SIZE", 40)
+DATABASE_URI = os.environ.get("DATABASE_URI","mysql+pymysql://2v2WvLxTSjY5adC.root:2rA6B5Zc98PgMqNH@gateway01.ap-southeast-1.prod.alicloud.tidbcloud.com:4000/test?ssl_ca=/etc/ssl/cert.pem&ssl_verify_cert=true&ssl_verify_identity=true")
```

Comment:
lol, now our database is leaked, please keep it in the configuration file or environment variable and do not commit